### DISCUSSION TITLE
Zoom with mouse wheel without CTRL

### DISCUSSION THREAD

**Discussion by PeterM32** [sumatrapdfreader/sumatrapdf#3054] [Q&A]
Hello,

is it possible to set sumatraPDF so that you can zoom with the mouse wheel without the additional key "CTRL".

That would be great. Thanks for your help.

Kind regards
Peter

**Comment by tactilis** [comment]
@PeterM32

How would you scroll?

**Comment by PeterM32** [comment]
Only with the mouse wheel (up = zoom in / down = zoom out)

**Comment by tactilis** [comment]
> Only with the mouse wheel (up = zoom in / down = zoom out)

??

In applications that handle documents (PDFs, .docx, .txt etc), the scroll wheel on the mouse is used to scroll up/down the document and CTRL+scroll_wheel is used to zoom.

What is the reason you want to change this convention in SumatraPDF?

If the mouse wheel was changed to zoom in/out, how would you scroll?

**Comment by PeterM32** [comment]
For that I use the hand tool ... just like in the Adobe Acrobat reader.

**Comment by PeterM32** [comment]
There is no possibility?

**Comment by GitHubRulesOK** [comment]
@PeterM32 
In some mices its possibe to "chord" thus holding both outer buttons may convert scroll to zoom.
**This works with my mouse in SumatraPDF** as well as other apps.

the scrolling is driven by OS hardware layer, to change mouse button/wheel behaviour is down to driver so one possibility is use a different driver hook at the hardware level and I always recommend [HiRez X-Mouse](https://www.highrez.co.uk/downloads/XMouseButtonControl.htm) as very versatile in handling per app situations

**Comment by PeterM32** [comment]
thank you...I'll try that once.

**Comment by PeterM32** [comment]
it works...great thanks "GitHubRulesOK" 👍😃

**Comment by sergeevabc** [comment]
In my case Ctrl+Scroll does not zoom. Perhaps this is due to the fact that instead of the mouse wheel, I use a notebook trackpoint?

**Comment by R1gg5** [comment]
Constantly mashing Ctrl (Control) on my Lenovo laptop seemed to lock this in. I'm using a BT KeyBoard. Forgot KB was asleep, hit CTRL and it did nothing so by the time I was done venting on the CTRL key and the mouse wheel, I was stuck with zooming instead of scrolling with the mouse wheel. Nothing worked, except for hitting the CTRL key several times with less frustration.

**Reply by GitHubRulesOK** [reply]
@R1gg5 
Sounds like auto sticky keys ?? on my keyboard that does not seem to work with control, but I may have set it to ignore ? as pressing or holding same key does not trigger it currently, as per here aaaaaaaaaaaaaaaaaaaaaaaa h!
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/5fc79307-1ae3-4d78-9ac8-5f8de5670b8e)

**Comment by ozbayseckin** [comment]
use right click and mouse wheel.

**Comment by JannetGen** [comment]
Totally agree, a dedicated dark mode toggle would be awesome. In the meantime, I’ve found that changing the BackgroundColor in the Advanced Settings file is the best workaround. Sumatra is so lightweight that I don't mind the 30 seconds it takes to tweak the config file.
